tests/Core/Tokenizer/BackfillFnTokenTest.inc

```diff
@@ -73,6 +73,49 @@ fn(array $a) : array => $a;
 /* testTernary */
 $fn = fn($a) => $a ? /* testTernaryThen */ fn() : string => 'a' : /* testTernaryElse */ fn() : string => 'b';
 
+/* testConstantDeclaration */
+const FN = 'a';
+
+class Foo {
+    /* testStaticMethodName */
+    public static function fn($param) {
+        /* testNestedInMethod */
+        $fn = fn($c) => $callable($factory($c), $c);
+    }
+
+    public function foo() {
+        /* testPropertyAssignment */
+        $this->fn = 'a';
+    }
+}
+
+$anon = new class() {
+    /* testAnonClassMethodName */
+    protected function fn($param) {
+    }
+}
+
+/* testNonArrowStaticMethodCall */
+$a = Foo::fn($param);
+
+/* testNonArrowStaticMethodCallWithChaining */
+$a = Foo::fn($param)->another();
+
+/* testNonArrowStaticConstant */
```

Comment:
This new test was added but is not checked anywhere. Should it be added to dataNotAnArrowFunction()?

Reply:
Resolved via #2863